package.json

```diff
@@ -32,9 +35,13 @@
     "react": "18.x"
   },
   "files": [
-    "dist"
+    "src"
   ],
   "scripts": {
+    "cosmos": "bun run cosmos:start & bun run cosmos:bun",
```

Comment:
encapsulate all of these into a single typescript file so we dont pollute package.json scripts